src/aioswitcher/api/__init__.py

```diff
@@ -811,10 +923,56 @@ async def get_shutter_state(self) -> SwitcherShutterStateResponse:
             self._writer.write(unhexlify(signed_packet))
             state_resp = await self._reader.read(1024)
             try:
-                response = SwitcherShutterStateResponse(state_resp)
+                response = SwitcherShutterStateResponse(
+                    state_resp, self._device_type, index
+                )
                 return response
             except (KeyError, ValueError) as ve:
                 raise RuntimeError(
                     "get shutter state request was not successful"
                 ) from ve
         raise RuntimeError("login request was not successful")
+
+    async def set_light(
+        self, command: LightState, index: int = 1
+    ) -> SwitcherBaseResponse:
+        """Use for turn on/off light.
+
+        Args:
+            command: use the ``aioswitcher.api.LightState`` enum.
+            index: which light to turn on/off, default to 1.
+
+        Returns:
+            An instance of ``SwitcherBaseResponse``.
+
+        """
+        index = get_light_index(self._device_type, index)
+        hex_pos = f"0{index}{command.value}" if index else command.value
+        precommand = "370a"
+
```

Comment:
It would be better if this will use a similar format we use for other commands.
https://github.com/TomerFi/aioswitcher/blob/dev/src/aioswitcher/api/packets.py

Reply:
@thecode see my comment above

Reply:
Done. You can resolve it.